src/main/java/org/mule/extensions/jms/internal/util/ActiveMQConnectionFactoryUtil.java

```diff
@@ -43,4 +47,25 @@ public static boolean isVerifyHostnameValidVersion(String activeMqClientVersion)
     }
     return patchVersion >= PATCH_VERSION_THRESHOLD;
   }
+
+  public static String brokerUrlFormat(String brokerURL, boolean verifyHostName) {
```

Comment:
A javadoc explaining what this method does would be reeeaaally helpful ❤️

Reply:
done